### ISSUE TITLE
Ability to hide taskbar

### ISSUE THREAD

**Issue by DNM1008** [dremin/RetroBar#77]
Like the title said, is there a way for the taskbar to hide/unhide like the ordinary taskbar can? Preferably with a shortcut

**Comment by rbreaves** [comment]
Upvoting this!!! Yes this app really needs this feature!

**Comment by ThaBluNate** [comment]
I second

**Comment by Oszku** [comment]
bump

**Comment by yereverluvinunclebert** [comment]
This is an ESSENTIAL and fundamental characteristic of the original taskbar, also mentioned in #469,  #319 and #346.

In my mind it is the ONLY important option that really needs to be fixed. This isn't just me moaning. I can live (at a push) without it. However, gaining that extra screen space is essential. My own dock has it, as one of the first things I implemented. 

Several requests for this now from different people. A timer that counts the last interaction with the dock, triggering another, this one providing a form opacity fade from 255 to 0 (and vice versa when the user's cursor enters the dock area)  - can't be that tricky to implement. 

Basically, if it doesn't hide it isn't acting like a proper dock should.  I could knock up some code if required, I am not a C# programmer but I have similar code in VB6.

**Comment by dremin** [comment]
I have been working on auto-hide support for RetroBar; here is a pre-release that can be tested. Please note that this has not gone through the full gamut of testing and may not be perfect!
[RetroBar.zip](https://github.com/dremin/RetroBar/files/11484142/RetroBar.zip)

**Comment by yereverluvinunclebert** [comment]
Tested on Windows 10 Pro 21H2. All working perfectly with a nice slide out. Stress testing now. 

Results checking on Windows 11 and 7 coming shortly.

**Comment by dremin** [comment]
This is available now in 1.13.
